src/middlewares/CheckIfModerator.js

```diff
@@ -0,0 +1,50 @@
+import models from '../api/models';
+import status from '../helpers/constants/status.codes';
+import sendError from '../helpers/error.sender';
+
+const { User } = models;
+/**
+ * check if user is admin
+ *
+ * @export
+ * @class CheckIfAdmin
+ */
+class CheckIfAdmin {
+  /**
+   * this is a middleware which checks if the user is admin.
+   *
+   * @static
+   * @param {object} req the request
+   * @param {object} res the response
+   * @param { object } next the next route controller to be called
+   * @memberof CheckUser
+   * @returns {Object} res
+   */
+  static async CheckAdmins(req, res, next) {
+    const {
+      user: { email }
+    } = req.user;
+    const admin = await User.findOne({
```

Comment:
@Ramadhan0 why not try a Sequelize `or` here instead of making 2 separate queries?

http://docs.sequelizejs.com/manual/querying.html#where